walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VCVerifiedIdSerializer.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved
+
+package com.microsoft.walletlibrary.verifiedid
+
+import com.microsoft.walletlibrary.did.sdk.credential.service.models.contracts.VerifiableCredentialContract
+
+object VCVerifiedIdSerializer : VerifiedIdSerializer<com.microsoft.walletlibrary.did.sdk.credential.models.VerifiableCredential> {
+    /**
+     * Serialize the given verifiedID into the SerializedFormat
+     */
+    override fun serialize(verifiedId: VerifiedId): com.microsoft.walletlibrary.did.sdk.credential.models.VerifiableCredential {
+        return when (verifiedId) {
+            is VerifiableCredential -> {
```

Comment:
When we have to deserialize this VerifiableCredential, how do we know what type of Verified ID it is? Like what if it is a different type of VerifiedID with different display information besides a contract like in Issuance V2?

Reply:
It wouldn't be possible with current schema. This will be used for converting the legacy VCs.